icechunk-python/benchmarks/lib.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass
+from typing import Any
+
+import fsspec
+
+import icechunk as ic
+
+
+@dataclass
+class Dataset:
+    bucket: str
+    prefix: str
+    storage: ic.Storage
+    chunk_selector: dict[str, Any]
+
+    def create(self) -> ic.Repository:
+        fs = fsspec.filesystem("s3")
+        try:
+            fs.rm(f"{self.bucket}/{self.prefix}", recursive=True)
+        except FileNotFoundError:
+            pass
+        return ic.Repository.create(self.storage)
+
+    @property
+    def store(self) -> ic.IcechunkStore:
+        repo = ic.Repository.open(self.storage)
+        return repo.readonly_session(branch="main").store
+
+
+ERA5 = Dataset(
+    # FIXME: make these accessible from storage
+    bucket="icechunk-test",
+    prefix="era5-demo-repository-a9",
```

Comment:
Tomorrow after Xarray & Zarr releases I'll remake this and `icebreakhrrr` and add them too.